.github/workflows/formatting.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     - name: 📐 Check code formatting
       run: |
-        nix develop .#fmt --command treefmt --fail-on-change
+        nix build .#checks.x86_64-linux.treefmt
```

Comment:
Should: avoid explicit system

Can we just leave this out?

Reply:
I prefer not to infer this. If it does generalise then it should parameterise via the matrix or so, but I think we assume to always run these checks on x86 linux.